### PR TITLE
Fix Atol doctest

### PR DESCRIPTION
**Pull request by mglisse**
the skip marks were on the wrong lines, which resulted in

> WARNING: ignoring invalid doctest code

Reminder: testoutput was disabled because it was too random (the lines are swapped if I run it right now).

### CODE REVIEW

File: src/python/gudhi/representations/vector_methods.py [hunk 1]
```diff
@@ -684,16 +684,16 @@ class Atol(BaseEstimator, TransformerMixin):
...
+    array([1.18168665, 0.42375966])
+    >>> atol_vectoriser(c) # doctest: +SKIP
+    array([0.02062512, 1.25157463])
     >>> atol_vectoriser.transform(X=[a, b, c]) # doctest: +SKIP
-    >>> # array([[1.18168665, 0.42375966],
-    >>> #        [0.29861028, 1.06330156],
-    >>> #        [0.02062512, 1.25157463]])
+    array([[1.18168665, 0.42375966],
+           [0.29861028, 1.06330156],
+           [0.02062512, 1.25157463]])
```
[the remark this answers is not in the document]
Comment (mglisse) [reply src/python/gudhi/representations/vector_methods.py:696]: IIUC, the example explicitly passes sklearn.cluster.KMeans as argument, and that class has "random" output, so it doesn't disturb me too much. But let's ask the author. @martinroyer , should we add some note in the doc to warn users that the output may vary?

[the remark this answers is not in the document]
Comment (mglisse) [reply src/python/gudhi/representations/vector_methods.py:696]: I think the "randomness" is more related to the version of sklearn (and possibly the architecture and OS of your computer), and not true randomness.

[the remark this answers is not in the document]
Comment (martinroyer) [reply src/python/gudhi/representations/vector_methods.py:696]: Perhaps a simple fix is to change Atol behavior like this:
Replace l. 692
        self.centers = self.quantiser.cluster_centers_
with
        self.centers = np.sort(self.quantiser.cluster_centers_, axis=0)

[the remark this answers is not in the document]
Comment (mglisse) [reply src/python/gudhi/representations/vector_methods.py:696]: I thought about that but it felt hackish. Still, in order to reduce variability, why not...

[the remark this answers is not in the document]
Comment (VincentRouvreau) [reply src/python/gudhi/representations/vector_methods.py:696]: I am ok either with a comment or the hack you propose, it is up to you @mglisse

[the remark this answers is not in the document]
Comment (mglisse) [reply src/python/gudhi/representations/vector_methods.py:696]: > `self.centers = np.sort(self.quantiser.cluster_centers_, axis=0)`

np.sort doesn't do the right thing here, it sorts each column independently whereas we would like a lexicographic sort of the lines.
I can get the test to output the string currently expected if I add `atol_vectoriser.centers=atol_vectoriser.centers[np.lexsort(atol_vectoriser.centers.T)][::-1]`
(ugly)

[the remark this answers is not in the document]
Comment (martinroyer) [reply src/python/gudhi/representations/vector_methods.py:696]: I think it's a decent fix!
